javascript/javascript1/week3/index.html

```diff
@@ -5,11 +5,18 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>JavaScript 1 - Week3</title>
+    <title>Smart-ease</title>
     <link rel="stylesheet" href="style.css">
 </head>
 
-<body>
+<body class="container-body">
+    <h1>Smart-ease</h1>
+    <h2>JavaScript1 - Week3 - Homework</h2>
+
+    <section>
+        <h3>Freecode Camp Exercise</h3>
+        <p>Username: Aditi-Sawardekar</p>
```

Comment:
Your free code camp is private , I can not see.

Reply:
Hi, My freecode camp profile is public. I have saved it again as public. Please check again